packages/web-client/app/utils/web3-strategies/layer2-chain.ts

```diff
@@ -230,15 +226,17 @@ export default abstract class Layer2ChainWeb3Strategy
   }
 
   async refreshBalances() {
-    return taskFor(this.fetchDepotTask).perform();
+    return taskFor(this.viewSafesTask).perform();
```

Comment:
Maybe current instances of `safes.fetch` can be replaced with a call to `refreshBalances`? To make that the standard interface.

Reply:
Good idea.